### ISSUE TITLE
Add issue template for inclusion of external projects into the `laminas/` project

### ISSUE THREAD

**Issue by Ocramius** [laminas/technical-steering-committee#5]
#### Summary

As discussed during today's TSC meeting, we need an issue template to allow people to request inclusion of third-party projects in the `laminas/*` project, and to add the maintainers of those projects as TSC members.

@Ocramius will provide the initial template PR.

**Comment by weierophinney** [comment]
*ahem*... @Ocramius :rofl:

**Comment by heiglandreas** [comment]
Erm.... This has not seen much action since... 1.5 years. Is this still relevant? Should we bring this onto the agenda of one of the next TSC-meetings?

**Comment by Ocramius** [comment]
yep, not really my pair of shoes here, so I'm adding `help needed` :P

**Comment by ALTAMASH80** [comment]
Hi, can anyone from the above can hear this person down here? I didn't understand the phrase **inclusion of external projects**. Also, which fields will be required for a template? Also, by template, you people mean the template like reporting issues. Sorry for being a code monkey. :smiling_face_with_tear:

**Comment by Ocramius** [comment]
@ALTAMASH80 the idea is that you (for example) may have created `altamash80/my-great-library` and want to move it into `laminas/*`

**Comment by ALTAMASH80** [comment]
@Ocramius, thanks for the answer. I've plans only to be helpful around building things for Laminas-MVC with doctrine. Need to first work on LmcUserDoctrine and LmcRbacMVC. These kinds of libraries aren't fit for `laminas/*`.
